cranelift-codegen/meta/src/isa/x86/encodings.rs

```diff
@@ -1992,6 +1985,21 @@ pub(crate) fn define(
         e.enc_32_64_maybe_isap(imul, rec_fa.opcodes(opcodes), *isap);
     }
 
+    // SIMD logical operations
+    for ty in ValueType::all_lane_types().filter(allowed_simd_type) {
+        // and
+        let band = band.bind_vector_from_lane(ty, sse_vector_size);
+        e.enc_32_64(band, rec_fa.opcodes(&PAND));
+
+        // or
+        let bor = bor.bind_vector_from_lane(ty, sse_vector_size);
+        e.enc_32_64(bor, rec_fa.opcodes(&POR));
```

Comment:
What is the difference between the old ORPS and POR?

Reply:
See the commit message on 62ad8de